osquery/tables/system/posix/sudoers.cpp

```diff
@@ -87,6 +103,11 @@ void genSudoersFile(const std::string& filename,
       continue;
     }
 
+    // Check if a blackslash is the last character on this line.
```

Comment:
This detection of a long line occurs after detection of an `include` or a comment, both of which will skip the detection of a long line. So I guess I'm wondering if a file could have lines like:

```
#
# This is a comment that is on a long line ending with a \
this is more comment but a continuation of the above line
# This is another comment line
```

or 

```
#includedir /etc/super/long/path/that/is/broken/by/a\ 
more/long/path/which/ends/on/this/line/sudoers.d
```

Reply:
@mike-myers-tob, No, this syntax is invalid. According of the manual of sudoers: The pound sign (‘#’) is used to indicate a comment or include file or directory directives. the pound sign is up to end-of-line and can't be continued in the next line with backslash. 
Moreover, you can copy both examples into a file and then execute the command: `visudo -c <file>` this is 'check-only' mode,  a supplied sudoers file will be checked for syntax errors. 
spoiler: you'll get a syntax error for your both examples.

Reply:
Excellent okay thanks for explaining